src/RemoteAPI/class-base-proxy.php

```diff
@@ -43,7 +43,7 @@ public function __construct( Parsely $parsely ) {
 	}
 
 	/**
-	 * Get URL for Recommendation API (GET /related).
+	 * Get URL for Recommendations API (GET /related).
```

Comment:
```suggestion
	 * Get URL for a particular Parse.ly API endpoint.
```

This base proxy class doesn't know anything about Recommendations or other endpoints.

Reply:
Good catch. Will also fix another comment in the class and push it in a single commit.